apps/src/templates/instructions/FeedbacksList.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {PropTypes, Component} from 'react';
+import i18n from '@cdo/locale';
+import moment from 'moment';
+
+const styles = {
+  content: {
+    padding: 10
+  },
+  header: {
+    fontWeight: 'bold',
+    paddingRight: 5
+  }
+};
+
+export default class FeedbacksList extends Component {
+  static propTypes = {
+    feedbacks: PropTypes.array,
```

Comment:
Since this component assumes `feedbacks` is present and interfaces directly with its contents, let's make the propTypes more specific if only for documentation.
```jsx
static propTypes = {
  feedbacks: PropTypes.arrayOf(PropTypes.shape({
    teacher_name: PropTypes.string.isRequired,
    created_at: PropTypes.string.isRequired,
    comment: PropTypes.string.isRequired
  })).isRequired
};
```
Am I correct that `created_at` is a string and `moment` interprets it?